app/password/page.tsx

```
@@ -0,0 +1,109 @@
+
+  return (
+    <>
+      <Navbar />
+
+      <div className="flex min-h-[calc(100vh-64px)] items-center justify-center px-4">
+        <div className="w-full max-w-md rounded-xl border bg-background p-6 shadow-sm">
+          <div className="mb-6 text-center">
+            <h1 className="text-2xl font-bold">Forgot Password</h1>
+
+            <p className="mt-2 text-sm text-muted-foreground">
+              Enter your email address and we'll send you a password reset link.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)